### DISCUSSION TITLE
Can’t paste content from outside terminal with Wezterm + Tmux

### DISCUSSION THREAD

**Discussion by 0xJWLabs** [wezterm/wezterm#6331] [Q&A]
### What Operating System(s) are you running on?

Windows

### WezTerm version

latest nightly

### Ask your question!

When using Wezterm in combination with Tmux, I am unable to paste content from outside sources (such as the system clipboard or other applications) into the terminal. This occurs regardless of the Tmux configuration, and affects productivity as it limits the use of external resources within Tmux sessions.

## Steps to Reproduce

	1.	Open Wezterm.
	2.	Start a Tmux session.
	3.	Attempt to paste content from an external application (e.g., copying text from a browser or editor and pasting it into the Tmux session).

## Expected Behavior
Text from outside the terminal (system clipboard) should be pasted into the Tmux session as expected.

**Comment by chenxin-yan** [comment]
I am experiencing a similar issue. When I paste multiline content from the clipboard inside a Tmux session, the content collapses into a single line and is not formatted properly.

**Comment by selfish** [comment]
Same as OP here. paste -> Nothing happens.

**Comment by bew** [comment]
You can tweak how newlines are normlized on paste on windows using [`canonicalize_pasted_newlines`](https://wezfurlong.org/wezterm/config/lua/config/canonicalize_pasted_newlines.html) config option.

**Comment by selfish** [comment]
@bew Thank you!
Notice that unlike @chenxin-yan's report, mine occurs also when no newlines are in the copied value, and I believe OP mentioned nothing related to newlines as well.
